Add case study: CW20 (#44)
softwaresaved/event-organisation-guide: docs/eog/case-studies-cw20-timeline.rst

.. _CW20-Timeline: 

CW20 Timeline
=================

We made the decision three weeks before CW20 to move the event online. 
We had less than one month to contact all of our stakeholders (venue, accommodation, sponsors, speakers, workshop facilitators and of course participants), inform them of the decision and renegotiate everything that had been previously agreed. 
As days passed and the implications around the global pandemic became more clear, stakeholders were very understanding and accommodating. 
Below is a breakdown of what we did. 

> Event Project Stage

Reviewer: The Event Project Stage is a container - it covers the stage when you get the big 'OK' to move forward with your event - so Pre-Event, During the Event and Post-Event are all contained in the Event Project Stage - so it's better to  put sub-headings in to reflect the sub-stages within the Event Project Stage.

Author: I have just removed those other headings, as heading levels in restructuredtext are not assigned to certain characters but the structure is determined from the succession of headings.

--------------------

Month -1 (March 2020)
^^^^^^^^^^^^^^^^^^^^^^

Agenda
  - We reorganised agenda for online format by
   

>     - Removing in-person social programme (walk in Belfast Botanical Gardens, visit to Ulster Museum, Drinks Reception, Conference Dinner).

Reviewer: This is the first time you mention that the event was originally meant to take place in Belfast (and perhaps it was worth mentioning that this would be the first CW taking place in Northern Ireland) - in the intro of the case study.

  - Contacted the event delivery team with new duties roster, held pre-event meetings to test the infrastructure and make sure everyone was comfortable with their role. 
  - In the days before the event, we sent emails to registrants through Eventbrite reminding them to review the Code of Conduct, instructions to install and test the Zoom software, and information about how to prepare for certain sessions. 
   
    - We also stressed that CW20 would be as flexible on the day as possible, as many people were adjusting to a new way of working, caring responsibilities during the day, unstable internet, and to the associated emotional pressures. 
    - We provided guidance for everyone to be kind, understanding and flexible. 
      
  - Links to the relevant Zoom room and note-taking document were sent via email through Eventbrite and Slack at the start of each day.
   
Resources
  - We set up a uCONFLY instance for management of Discussion Group Reports, Mini-workship and Demo session notes, Collaborative Ideas and Hack Day ideas which also involved
   
    - Creating Google Doc templates for each of these document types.

>     - Generating a number of each document type for dissemination during CW20 so that participants would not be responsible for yet another platform.

Reviewer: ```suggestion
    - Generating a number of each document type for dissemination during CW20 so that participants would not have to login and use yet another platform.
```

      
  - Built the infrastructure needed to synchronise participants and run the unconference sessions such as
   
    - Collaborative notes documents with agenda and relevant links.
    - Spreadsheets to disseminate the uCONFLY generated documents. 
      
  - Wrote documentation for the delivery team including
   
    - Zoom instructions 
    - Duties roster
    - Links index
    - Start of day tasks for Event Lead/Chair
    - Mini-workshop and demo session assignments
      
  - Made organiser slide deck.
  
Event run up actions
  - Closed registration immediately upon decision to move event online.

>   - Issued full refunds to registrants who were no longer able to attend up to 7 days before the event start date.

Reviewer: ```suggestion
  - Issued full refunds to registrants who were no longer able to attend and informed us at least 7 days before the event start date.
```

  - Issued partial refunds for the difference between the original priced and the new reduced price to registrants who still wanted to attend.
  - Planned break-out groups distribution (sign up sheet for Discussion Groups session, randomly assigned for Collaborative Ideas session).
  - Planned for video recording infrastructure (easy recording of sessions via Zoom).
  - Had a Code of Conduct committee pre-event meeting.
  - Constructed feedback form draft.

Event Running Stage
--------------------

> Month 0

Reviewer: Might be worth saying it ran smoothly according to plan and any funnies that happened - I don't know if making everyone co-host to allow mingling for the hackday team formation is worth mentioning and that is an area we hope to improve in the future - i.e. mingling in online workshops.

  - Published a `participant's experience of CW20 <https://software.ac.uk/blog/2020-04-29-ssi-collaborations-workshop-2020-remote-unconference-experience-and-notes>`_.
  
Resources
  - Updated language of event website to show that it was in the past.
  - Created a `Figshare portal <https://cw20.figshare.com/>`_ to share CW20 outputs
  - Chased speakers for any un-captured resources.
  - Collected Zoom recordings from the Hosts/Co-Hosts of each session.
  - Prepared videos for editing by
   
    - Organising files.
    - Updating processing instructions for Events Team.
      
  - Followed up with speed-blog authors and connected them with the Communications Team for publication to the SSI blog.

Month 2 (May 2020)
^^^^^^^^^^^^^^^^^^^^^^

Publicity
  - Published a reflections blog post about `how we moved the event online in three weeks <https://software.ac.uk/blog/2020-05-18-cw20-how-move-event-online-three-weeks>`_.
  
Resources
  - 9/12 speed blogs published.

Month 3 (June 2020)
^^^^^^^^^^^^^^^^^^^^^^

Publicity
  - Announced availability of `Infrastructure resources <https://software.ac.uk/news/collaborations-workshop-2020-resources-now-available>`_.
  
Resources

>   - 10/12 speed blogs published.

Reviewer: Is is the case that all 12 will be published - or were there 12 sessions - and 10 made it as published blogs of the right quality (at the moment it reads as more maybe coming).

  
Resources
  - 9/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_.

Month 3 (June 2020)
^^^^^^^^^^^^^^^^^^^^^^

Publicity
  - Announced availability of `Infrastructure resources <https://software.ac.uk/news/collaborations-workshop-2020-resources-now-available>`_.
  
Resources
  - 10/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_.
  - Infrastructure resources sanitised and `shared <https://doi.org/10.6084/m9.figshare.c.5026400.v3>`_.

Month 4 (July 2020)
^^^^^^^^^^^^^^^^^^^^

Publicity
  - Announced availability of `session recordings <https://software.ac.uk/news/collaborations-workshop-2020-session-recordings-now-available>`_.
  
Resources
  - 100% of videos processed and `uploaded <https://www.youtube.com/playlist?list=PLpX1jXuNTXGoVHrK6IPrn4Ymwaj6l4KLJ>`_.

Month 5 (August 2020)
^^^^^^^^^^^^^^^^^^^^

- 11/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_ - the last one may be published in the future if the group members decide to.

> 

Reviewer: Do we want a Month 6 (September 2020)

+ EOG case study completed and reviewed
